wire-ios/WireCommonComponents/Analytics/WireAnalytics.swift

```diff
@@ -43,12 +43,56 @@ public enum WireAnalytics {
             #if DEBUG
                 SystemLogger()
             #endif
-            CocoaLumberjackLogger()
+            CocoaLumberjackLogger(logsDirectory: target.logsDirectory)
             WireAnalytics.Datadog.shared
         }
 
         // pass tags to Datadog through WireLogger
         WireLogger.system.addTag(.processId, value: "\(ProcessInfo.processInfo.processIdentifier)")
         WireLogger.system.addTag(.processName, value: ProcessInfo.processInfo.processName)
     }
+
+    public enum Target {
+        case app
+        case notificationServiceExtension
+        case shareExtension
+    }
+
+}
+
+extension WireAnalytics.Target {
+
+    fileprivate var logsDirectory: URL? {
+        let fileManager = FileManager.default
+        switch self {
+
+        case .app:
+            let cachesDirectory = try? fileManager.url(
+                for: .cachesDirectory,
+                in: .userDomainMask,
+                appropriateFor: nil,
+                create: true
+            )
+            return cachesDirectory?.appending(path: "Logs", directoryHint: .isDirectory)
```

Comment:
todo: I think we should update LogFilesProvider to use this location too then. So it's defined only in one place.
To answer your question about moving also the main app logs to shared group, we can do it, but we have to make sure we clean up:
- after sending logs 
- after logout